src/supervisor2.erl

```diff
@@ -1085,7 +1085,7 @@ monitor_child(Pid) ->
 %%
 %% Shutdown all dynamic children. This happens when the supervisor is
 %% stopped. Because the supervisor can have millions of dynamic children, we
-%% can have an significative overhead here.
+%% can have significant overhead here.
```

Comment:
I'd never seen this word before, it's apparently a word, that said, it almost certainly wouldn't be compatible w/ `an`, and I don't think it's the right word...

Reply:
I agree, the intent is to say "significant overhead".